aws_ami/config/ami-automl.pkr.hcl

```diff
@@ -0,0 +1,51 @@
+variable "source_ami" {
+  type    = string
+  description = "Ubuntu Server 18.04 LTS (HVM), EBS General Purpose (SSD) VolumeType"
+  default = "ami-0bdf93799014acdc4"
```

Comment:
this defaults to an ami available only in `eu-central-1`: would be nice to have a way to automatically default to the ami defined associated to the selected region in `config.yaml`.